src/main/webapp/js/feedbackResponseComments.js

```diff
@@ -43,6 +43,7 @@ var addCommentHandler = function(e) {
 	                    var newCommentRow = addFormRow.prev();
 	                    newCommentRow.find("form[class*='responseCommentEditForm'] > div > a[id*='button_save_comment_for_edit']").click(editCommentHandler);
 	                    newCommentRow.find("form[class*='responseCommentDeleteForm'] > a").click(deleteCommentHandler);
+	                    registerCheckboxEventForVisibilityOptions();
```

Comment:
this register of event is for student comment.
try the regression yourself, then u will know it